mln/services/message.py

```diff
@@ -1,5 +1,5 @@
-from ..models.dynamic import FriendshipStatus, Message
-from ..models.static import MessageBody
+from ..models.dynamic import Attachment, FriendshipStatus, Message
+from ..models.static import MessageBody, NetworkerReplyTrigger

```

Comment:
This seems like it was left in by mistake?

Reply:
For the next PR (#12), I guess